doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -0,0 +1,253 @@
+(kuberay-gcs-ft)=
+# GCS fault tolerance in KubeRay
```

Comment:
Is this a new doc or have we just moved it from the kuberay repo?

Reply:
KubeRay originally had one, but I have rewritten everything.